incubator/hnc/Makefile

```diff
@@ -128,14 +135,14 @@ endif
 # deleting the CRDs will cause all the existing hierarchy configs to be wiped
 # away and b) if we don't delete the deployment, a new image won't be pulled
 # unless the tag changes.
-deploy: deploy-prereq docker-push kubectl manifests
+deploy: docker-push kubectl manifests
 	-kubectl -n hnc-system delete deployment hnc-controller-manager
 	kubectl apply -f manifests/hnc-manager.yaml
 
 deploy-watch:
 	kubectl logs -n hnc-system --follow deployment/hnc-controller-manager manager
 
-# Installs prerequisites. See https://cert-manager.io/docs/installation/kubernetes/ for why we're not validating our YAML.
+# Installs external certs before deploying HNC. See https://cert-manager.io/docs/installation/kubernetes/ for why we're not validating our YAML.
```

Comment:
Maybe change to deploy-cm, since that's now all it does? They're not really prereqs anymore